### PR TITLE
feat(release-trigger): Add GoogleCloudDataproc to allowlist

### PR DESCRIPTION
**Pull request by bsidhom**
Adding the Dataproc organization to the release-trigger allowlist.

### CODE REVIEW

File: packages/release-trigger/src/release-trigger.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ export const execFile = function (
   });
 };
 
-export const ALLOWED_ORGANIZATIONS = ['googleapis', 'GoogleCloudPlatform'];
+export const ALLOWED_ORGANIZATIONS = ['googleapis', 'GoogleCloudDataproc', 'GoogleCloudPlatform'];
```
Comment (chingor13) [inline packages/release-trigger/src/release-trigger.ts:66]: Looks like lint is failing.

Can you make the suggested change from below? (or run `npm run fix`)

**Review by chingor13 — CHANGES_REQUESTED** [review]

File: packages/release-trigger/src/release-trigger.ts (hunk 1 above)
Comment (bsidhom) [reply packages/release-trigger/src/release-trigger.ts]: Done.

**Review by chingor13 — APPROVED** [review]

**Pull request merged**